war/src/main/js/add-item.js

```diff
@@ -116,6 +116,18 @@ $.when(getItems()).done(function(data) {
       }
     }
 
+    function focusButton(status) {
+      var btn = $('form .footer .btn-decorator button[type=submit]');
+      if (status === true) {
+        btn.focus();
+      }
```

Comment:
Also ... as things are, there's no need to do the jquery select until inside the `if` block i.e. at the moment it's redundant what `status === false`.